### ISSUE TITLE
It doesn't seem possible to load two visualizations in the same notebook.

### ISSUE THREAD

**Issue by moorepants** [pydy/pydy#218]
Say you build a model, visualize it in the notebook, and then modify the model in cells below the viz, and then finally visualize again. This causes issues. The second viz gets a 404 on the scene json file and the first viz breaks. Restarting the kernel locks the notebook. Bad stuff. It would be nice to have multiple viz widgets in one notebook. This was is problem for the human standing tutorial: https://github.com/pydy/pydy-tutorial-human-standing/issues/78

**Comment by tarzzz** [comment]
This *might* be because the javascripts are getting loaded twice, inside the same notebook. 
Loading the same javascript libraries multiple times inside a single html page causes issues ( a known issue), 
If something like this is desired, we should probably figure out a way to check whether the libraries are already loaded, and do not load them again.

**Comment by moorepants** [comment]
Thanks for that. Yes, I think you may want to have two visualization in one notebook. You may create two models in a notebook or two simulations of one model, for example.

This brings up whether we are using the IPython JS stuff correctly. I feel like I read that the recommended practice is to load in custom JS code through an ndextension.

**Comment by tarzzz** [comment]
By the way, noticed a very similar issue in mass_spring_damper example. When we run the visualization for the first time, it works fine. Then edit some python code in the cells above(without restarting kernel), and execute `display_ipython` while the original visualization is already there, it also leads to loading the JS twice. The notebook freezes, and kernel needs to be restarted to continue working. 

Sometimes it works when run twice, and freezes when `display_ipython` is run three times in a row.

**Comment by moorepants** [comment]
Yes, this is a bit problematic. It would be nice to have this issue fixed for the release.
